src/v/pandaproxy/schema_registry/json.cc

```diff
@@ -1798,11 +1798,48 @@ bool is_positive_combinator_superset(
         break;
     }
 
-    throw as_exception(invalid_schema(fmt::format(
-      "{} not implemented. input: older: '{}', newer: '{}'",
-      __FUNCTION__,
-      pj{older},
-      pj{newer})));
+    // size are compatible, now we need to check that every schema from
+    // the smaller schema array has a unique compatible schema.
```

Comment:
Is "smaller schema array" correct in this comment?

Reply:
yes.

for allOf, newer can have more subschemas than older (each new subschema is more rules)
for oneOf/anyOf, newer can have less subschemas than older (any subschema removed reduces the degrees of freedom) 
the algo tries to do a match and we are concerned to ensure that the smaller subschema array is covered